src/componets/qoutes.js

```diff
@@ -0,0 +1,43 @@
+/*eslint-disable*/
```

Comment:
Consider enabling the linter to perform checks on this file in order for it to pick up the syntax errors in your code